oauth2_http/java/com/google/auth/oauth2/ComputeEngineCredentials.java

```diff
@@ -643,6 +657,9 @@ public static class Builder extends GoogleCredentials.Builder {
     private Collection<String> scopes;
     private Collection<String> defaultScopes;
 
+    private String transport = "";
+    private String bindingEnforcement = "";
```

Comment:
nit:
```suggestion
    private String transport;
    private String bindingEnforcement;
```

Reply:
Done in https://github.com/googleapis/google-auth-library-java/pull/1558/commits/3cb6e9852e77c77944d1418eec03bb5fdd970760